pkg/apis/networking/v1/utils.go

```diff
@@ -56,6 +57,21 @@ func GetNetworkType(networkObj *Network) NetworkType {
 	return networkObj.Spec.Type
 }
 
+func GetNetworkMode(networkObj *Network) NetworkMode {
+	switch GetNetworkType(networkObj) {
```

Comment:
if networkObject is nil, no valid network mode should be returned.

Reply:
For me, it just depends the behaviors of `GetNetworkType` here, if `GetNetworkType` always get a network type, maybe it is also reasonable to always return a network mode here.